aiida_quantumespresso/cli/workflows/q2r/base.py

```diff
@@ -12,7 +12,7 @@
 
 @cmd_launch.command('q2r-base')
 @options.CODE(required=True, type=types.CodeParamType(entry_point='quantumespresso.q2r'))
-@options.CALCULATION(type=types.CalculationParamType(sub_classes=('aiida.calculations:quantumespresso.ph',)))
+@options.CALCULATION()
```

Comment:
Idem -- This should actually be:
```suggestion
@options.CALCULATION(required=True)
```
And same comment about checking the type.

Reply:
will add type check and make it required